jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/nodes/CloudWatchRootNode.kt

```diff
@@ -0,0 +1,16 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.explorer.nodes
+
+import com.intellij.openapi.project.Project
+import software.amazon.awssdk.services.cloudwatchlogs.CloudWatchLogsClient
+import software.aws.toolkits.jetbrains.services.cloudwatch.logs.CloudWatchLogsServiceNode
+import software.aws.toolkits.resources.message
+
+class CloudWatchRootNode : AwsExplorerServiceNode {
```

Comment:
sidenote: Maybe we should get rid of the root node classes and have the extension points point directly to the service root node that gets constructed here

Reply:
Yep we might be able to refactor it that way, but there was a comment on that PR about how it wasn't so easy

Reply:
Fair, it just feels like a lot of classes that do nothing but wrap another class